README.md

```diff
@@ -296,3 +296,29 @@ let save text =
 #### Can I bind to events and use behaviors?
 
 Sure! Check out the [EventBindingsAndBehaviors sample](https://github.com/elmish/Elmish.WPF/tree/master/src/Samples). Note that you have to install the NuGet package `Microsoft.Xaml.Behaviors.Wpf`.
+
+#### How can I control logging?
+
+Elmish.WPF uses `Microsoft.Extensions.Logging`. To see Elmish.WPF output in your favorite logging framework, use `WpfProgram.withLogger` to pass an `ILoggerFactory`:
+
+```f#
+WpfProgram.mkSimple init update bindings
+|> WpfProgram.withLogger yourLoggerFactory
+|> WpfProgram.runWindow window
+```
+
+For example, in Serilog, you need to install Serilog.Extensions.Logging and instantiate `SerilogLoggerFactory`. The samples demonstrate this.
+
+Elmish.WPF logs to these categories:
+
+* `Elmish.WPF.Update`: Logs exceptions (Error level) and messages/models (Trace/Verbose level) during `update`.
```

Comment:
Since `Microsoft.Extensions.Logging` calls it `Trace`, can we stick to that naming?

```suggestion
* `Elmish.WPF.Update`: Logs exceptions (Error level) and messages/models (Trace level) during `update`.
```

Reply:
I think it's fine since our samples use Serilog and it's called `Verbose` there. Each sample has no references to `Trace` and three references to `Verbose`. But if you feel strongly about it, we can change it.

Reply:
I don't feel strongly, so whatever you want is fine.

Reply:
Let's keep it like this, then.